### DISCUSSION TITLE
Scroll bar arrows

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1208] [Team Posts]
Hi guys,
Do you have an opinion on the scroll bar arrows?

![image](https://user-images.githubusercontent.com/10009541/112375644-cbd30080-8ce3-11eb-82c4-7656f41eaaa2.png)

I had to theme the scroll bars as they looked too inconsistently on different desktops without theming. The blue arrows seem quite nice, but I'm not sure if they are not too unusual or too colorful. Options are to make them grey or too remove them completely (like I did in the Deep ocean theme and like many current apps do).

**Comment by spamatica** [comment]
I have no strong opinion either way :o)

Den ons 24 mars 2021 kl 21:03 skrev kybos ***@***.***>:

**Comment by terminator356** [comment]
Me neither. But no arrows sounds a bit odd. No precise consistent stepping?

**Comment by kybos** [comment]
Thanks guys, that was helpful, shows me that it's not as crazy as I sometimes thought.
No arrows: I agree that it's not the best idea, it's just that I'm seeing that in more and more apps (e.g. this Firefox I'm just in, but maybe it's the system theme?).
